modules/react/common/spec/createCanvasTheme.spec.tsx

```diff
@@ -1,13 +1,25 @@
 import colors from '@workday/canvas-colors-web';
-import {defaultCanvasTheme, createCanvasTheme, PartialCanvasTheme} from '../lib/theming';
+import {
+  defaultCanvasTheme,
+  createCanvasTheme,
+  PartialCanvasTheme,
+  getObjectProxy,
+} from '../lib/theming';
 import {shiftColor, ColorDirection} from '../lib/theming/createCanvasTheme';
-import lodash from 'lodash';
+
+const validateDefaultThemeProps = (theme: PartialCanvasTheme) => {
+  expect(theme.direction).toEqual(defaultCanvasTheme.direction);
+  expect(theme.palette.alert.dark).toEqual(defaultCanvasTheme.palette.alert.dark);
+  expect(theme.palette.common.focusOutline).toEqual(defaultCanvasTheme.palette.common.focusOutline);
+  expect(theme.breakpoints.values).toEqual(defaultCanvasTheme.breakpoints.values);
+};
 
 describe('createCanvasTheme', () => {
   test('calling without any input provides the default theme', () => {
     const theme = createCanvasTheme({});
 
-    expect(theme).toEqual(defaultCanvasTheme);
+    // Since createCanvasTheme is a proxy we need to access the properties directly. I don't particular want to check every property so checking only some should be enough
```

Comment:
This I am unsure about, I know createCanvasTheme is temporary so didn't want to spend a huge amount of effort going through each element in the proxy

Reply:
that's ok, i think it will be replaced with what's in getCanvasTheme